### PR TITLE
Update EFDT library to improve accessibility

### PR DESCRIPTION
**Pull request by zhx828**
# What? Why?
Relates to https://github.com/cBioPortal/enhanced-fixed-datatable/pull/34

# Checks
- [x] Runs on Heroku.
- [x] Single commit and [No merge commits](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [x] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)
- [x] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to
  hotfix.

# Any screenshots or GIFs?
No

# Notify reviewers
@fedde-s @pieterlukasse 

@fedde-s please notice that class and tabindex are not supported, I changed to className and tabIndex

### CODE REVIEW

**Comment by fedde-s** [comment]
> please notice that class and tabindex are not supported,
> I changed to className and tabIndex

Ah, I understand that using these DOM property names in the React code (XML in the JSX, compiled to `createElement()` calls in the JS) will be translated to the HTML attributes `class` and `tabindex` in the resulting HTML?

Looks like it works on my system: jQuery UI recognises the `.ui-slider-handle` elements and the accessibility issue is fixed 👍

File: portal/src/main/webapp/js/src/study-view/component/MutatedGeneCNATable.js [hunk 1]
```diff
@@ -218,6 +218,14 @@ var EnhancedFixedDataTableSpecial = (function() {
...
                     hideColumns(list);
                 });
+            // add title attributes to unlabeled inputs generated by
+            // .dropdownCheckbox() for accessibility, until
+            // https://github.com/Nelrohd/bootstrap-dropdown-checkbox/issues/33 is
+            // fixed upstream
+            $('#hide_column_checklist input[type="checkbox"].checkbox-all')
+                .attr('title', 'Select all');
+            $('#hide_column_checklist input[type="text"].search')
+                .attr('title', 'Search');
```
Comment (fedde-s) [inline portal/src/main/webapp/js/src/study-view/component/MutatedGeneCNATable.js]: @zhx828: may I ask how you updated this file? I do not see this file in the cBioPortal/enhanced-fixed-datatable repository. In pull request #1494 I proposed some minor changes to parts of this file that are specific to cBioPortal, did I apply those in the right place?

Comment (zhx828) [inline portal/src/main/webapp/js/src/study-view/component/MutatedGeneCNATable.js]: Yes, you did! I think these changes also will benefit the general EFDT library and that's why I asked you sent to the master branch. We have two cbioportal specific designed branches study-view-summary and study-view-clinical. Everytime if any changes made to master, I will rebase these two branches and update the library in our cbioportal repository.

**Comment by zhx828** [comment]
And yes @fedde-s, React will transfer that to related HTML attributes. React has a list of supported HTML attributes. Please see [here](https://facebook.github.io/react/docs/tags-and-attributes.html#html-attributes)
All attributes are camel-cased and the attributes class and for are className and htmlFor, respectively, to match the DOM API specification.

**Pull request merged**